modules/manage/partials/kubernetes/mtls.adoc

```diff
@@ -1,23 +1,157 @@
 [[mtls]]
-= Configure mTLS
+== Configure mTLS encryption
 
 Mutual TLS (mTLS), is an enhanced security protocol that extends the standard TLS encryption protocol. While standard TLS involves a server presenting a certificate to the client to prove its identity, mTLS adds an additional layer of security by requiring the client to also present a certificate to the server.
 
-To enable mTLS, set the `requireClientAuth` setting for a listener to `true`.
+To enable mTLS, set the `requireClientAuth` setting to `true` for all listeners.
```

Comment:
@chrisseto is this all listeners of the same name? For example, the snippet enables mTLS for all internal listeners and not the external ones.